From c350a8b374e8d4571f828dc71673666caec2633c Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Mon, 24 Nov 2025 20:11:38 +0100
Subject: [PATCH] CMakeLists: update min version to 3.10 for CMake >= 4.0
 support

Update minimum version of CMake to 3.10 for CMake >= 4.0 version
support.

New CMake require 3.5 as the minimum version with that increased to 3.10
in the next CMake release.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.10)
 project(ratechecker)
 
 set( CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/bin" )
